shared/templates/template_OVAL_file_owner

```diff
@@ -0,0 +1,25 @@
+<def-group>
+  <definition class="compliance" id="file_owner{{{ FILEID }}}" version="1">
+    <metadata>
+      <title>Verify {{{ FILEPATH }}} Owner Permissions</title>
```

Comment:
Drop Permissions from tittle, this OVAL is checking the file owner, not the file owner permissions.